src/main/java/com/synopsys/integration/jenkins/scan/exception/NoStackTraceException.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * synopsys-security-scan-plugin
+ *
+ * Copyright (c) 2023 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.jenkins.scan.exception;
+
+public class NoStackTraceException extends Exception {
```

Comment:
_`com.synopsys.integration.jenkins.scan.exception.NoStackTraceException: Workflow failed! Exit Code: 2 Error from adapter`_

If we go with this Exception way (not jenkins build failure way), then maybe we need to rename this `NoStackTraceException` class name. This name doesn't suit in client side.